editor/queries/crashNotes.ts

```diff
@@ -2,12 +2,13 @@ import { gql } from "graphql-request";
 
 export const INSERT_CRASH_NOTE = gql`
   mutation InsertCrashNote(
-    $crashPk: Int!
+    $recordId: Int!
     $text: String!
     $userEmail: String!
+    $createdBy: String!
```

Comment:
i think this `createdBy` var can be removed. i am hitting a graphql error when i try to create a _crash_ note because it's missing.

Reply:
thanks i missed this!